examples/stencil/src/components/todo-item/todo-item.tsx

```diff
@@ -0,0 +1,96 @@
+import { Component, h, State, Prop } from "@stencil/core";
+import { todomvc } from '../../store';
+
+// ---
+
+todomvc.on('todo:toggle', (state, id) => {
+	state.todos = state.todos.map(x => {
+		if (x.id === id) x.completed = !x.completed;
+		return x;
+	});
+});
+
+todomvc.on('todo:del', (state, id) => {
+	state.todos = state.todos.filter(x => x.id !== id);
+});
+
+todomvc.on('todo:put', (state, data) => {
+	state.todos = state.todos.map(x => {
+		if (x.id === data.id) x.title = data.title;
+		return x;
+	});
+});
+
+// ---
+
+@Component({
+	tag: "todo-item",
+})
+export class TodoItem {
+  @State() editing: boolean = false;
+  @Prop() _id: Todo['id'];
+  @Prop() completed: Todo['completed'];
+  @Prop() _title: Todo['title'];
```

Comment:
`title` and `id` are both reserved attributes for HTML elements and so they can't be used as prop names. I thought this was a simple workaround.

Reply:
Nice; feel free to renames these to `uid` + `text` if you want. I've not used Stencil so I'm not sure how common this kind of workaround is.

Reply:
Nothing is really common in Stencil at this point haha, but sounds good, I'll make that change

Reply:
I only renamed things in this component and left the rest the same. If you meant for me to change it everywhere, I'm on the fence about change the data structure for the global state.

I'm leaning towards keeping it the vegemite-specific stuff consistent with the other examples for the sake comparability and maintainability (if vegemite changes, it makes it easier to copy/paste between examples).